base/applications/osk/settings.c

```diff
@@ -16,13 +16,14 @@ BOOL LoadDataFromRegistry()
 {
     HKEY hKey;
     LONG lResult;
-    DWORD dwShowWarningData, dwLayout, dwSoundOnClick, dwPositionLeft, dwPositionTop;
+    DWORD dwShowWarningData, dwLayout, dwSoundOnClick, dwPositionLeft, dwPositionTop, dwAlwaysOnTop;
```

Comment:
Idea for new PR: no need to have one DWORD per registry value to read: a single one "dwValue" is sufficient, and you use it every time you load a value from the registry, that you then convert into a suitable value for the Globals variables.